AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -109,9 +151,50 @@ class LiveTranscriptionOptionsViewController: UIViewController, UITextFieldDeleg
         "us-west-2": "United States (Oregon)"
     ]
     
+    let partialResultsStabilizationsList = [
+        "Partial Results Stabilization",
+        "-- DEFAULT (HIGH) --",
+        "Low",
+        "Medium",
+        "High"
+    ]
+    
+    var piiContentIdentificationDict = [
```

Comment:
why do we have a list and a map storing duplicative data?

Would a 2-string data class work here instead?

Reply:
the drop down menu's contents are loaded in from the list. I can use just the map, but then the drop down's content's would be set as `Array(piiContentIdentificationDict.keys)`. Since the dictionary keys aren't sorted, the selection menu items will be in a random order. Some things we want at the top, (selection label, "ALL"), and would also want to keep consistency on how the items are displayed across platforms.

Reply:
got it. lets keep list to maintain ordering.